Jenkinsfile_CNP

```diff
@@ -60,19 +60,13 @@ def publishRpaContract() {
   sh """ ./gradlew pactPublish """
 }
 
-def checkForEnableHelmLabel(branch_name) {
```

Comment:
no need to delete it,- we can just change the name of the label

Reply:
I agree with you.
I back with this part of the code.